### ISSUE TITLE
create_function() should be a blocker

### ISSUE THREAD

**Issue by emrikol** [Automattic/VIP-Coding-Standards#2]
It's currently at a "warning" level:

`create_function is discouraged, please use Anonymous functions instead.`

**Comment by david-binda** [comment]
I'm not quite sure whether the Blocker level is appropriate. It is listed as a blocker in https://vip.wordpress.com/documentation/vip/code-review-what-we-look-for/#eval-and-create_function but the message is not that strict:

> If there are no other options than to use this construct, pay special attention not to pass any user provided data into it without properly validating it beforehand.

The PHPCS WordPress Coding Standards were fine tuned for the create_function in https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/commit/27ce2f8ad362e588540e7232a64613214442fba9 which separates the `eval` from `create_function`.

If we agree that `create_function` should be an Error (aka Blocker) and not a Warning, then we should report the issue in the upstream repository.

**Comment by rebeccahum** [comment]
It's listed as a warning for VIP Go though: https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/#eval-and-create_function

**Comment by rebeccahum** [comment]
It's listed as a warning for VIP Go though: https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/#eval-and-create_function so should it remain as a warning for VIP Go and listed as a blocker upstream?

**Comment by GaryJones** [comment]
As of PHP 7.2, `create_function` is deprecated and highly discouraged.

For the sake of platform capability, I think this should be marked as an Error.

**Comment by rebeccahum** [comment]
Looks like this is at  an error level?

```
--------------------------------------------------------------------------------
 3 | ERROR | create_function() is deprecated as of PHP 7.2.0
   |       | (WordPressVIPMinimum.Functions.CreateFunction.CreateFunction)
```

**Comment by rebeccahum** [comment]
This appears to be at an error level now:

> --------------------------------------------------------------------------------
> 3 | ERROR | create_function() is deprecated as of PHP 7.2.0
>  |       | (WordPressVIPMinimum.Functions.CreateFunction.CreateFunction)

**Comment by GaryJones** [comment]
> This appears to be at an error level now

You're right: [`addError()` is called](https://github.com/Automattic/VIP-Coding-Standards/blob/f7063ae564578efcc31189fb172472b9dc46484e/WordPressVIPMinimum/Sniffs/Functions/CreateFunctionSniff.php#L82), and it's not overwritten in either of the rulesets.
